src/Color.cpp

```diff
@@ -43,6 +43,27 @@ Rgb& Rgb::operator=( Hsv hsv ) {
     return *this;
 }
 
+Rgb Rgb::operator+( Rgb in ){
+    uint16_t red = this->r + in.r;
```

Comment:
Why do you use `this->`? The rest of the library does not use it. Be consistent, please.
Also, why do you choose uint16_t? Unless it is necessary, I would prefer platform default type (`int` and `unsigned int`) as they are big enough and their usage could yield a more efficient code.